Automattic/wp-calypso
# Add block pattern dom validation tests

#### Changes proposed in this Pull Request

* Add some automatic tests for block-patterns

#### Testing instructions

```
cd apps/editing-toolkit/editing-toolkit-plugin/
yarn run test:php
```

Check the [README](https://github.com/Automattic/wp-calypso/blob/master/apps/editing-toolkit/README.md) for setup instructions

In addition to verifying that these tests run green with the current patterns, it would also be good to check that they fail with broken patterns by editing some of the patterns in the `block-patterns/patterns` directory, as sometimes the tests fail silently (You might also want to run `npx wp-env run phpunit 'phpunit --debug -c /var/www/html/wp-content/plugins/editing-toolkit-plugin/phpunit.xml.dist'` in `apps/editing-toolkit` to verify that the couple of pages as the new test_patterns_pass_dom_validation runs on each pattern.)

In particular, it should warn about the missing closing double-quotes that started this off in #43817, as well as catching generally broken html (incorrect/unbalanced tags etc).
